### PR TITLE
Satisfy Clippy

### PR DESCRIPTION
**Pull request by Urhengulas**

### CODE REVIEW

File: firmware/qemu/src/bin/dbg.rs [hunk 1]
```diff
@@ -20,9 +21,11 @@ fn main() -> ! {
     let _: (i32, i32) = dbg!(x - 2, x + 2);
 
     // dbg! with zero arguments
+    #[allow(clippy::let_unit_value)]
     let _: () = dbg!();
```
Comment (jonas-schievink) [inline firmware/qemu/src/bin/dbg.rs:25]: This line could instead be `let () = dbg!();`, which is a bit cleaner and might also silence the warning

File: firmware/qemu/src/bin/dbg.rs [hunk 2]
```diff
@@ -20,9 +21,11 @@ fn main() -> ! {
...
 
     // dbg! with trailing comma
+    #[allow(clippy::double_parens)]
```
Comment (jonas-schievink) [inline firmware/qemu/src/bin/dbg.rs:28]: I think this is a bug in clippy: https://github.com/rust-lang/rust-clippy/issues/9000

File: firmware/qemu/src/bin/dbg.rs (hunk 1 above)
Comment (Urhengulas) [reply firmware/qemu/src/bin/dbg.rs:25]: Kk. But it sadly does not silence the warning.

**Comment by jonas-schievink** [comment]
CI fails due to https://github.com/la10736/rstest/issues/169 – I'm not sure why our nightly runs are blocking merges though, we might not want to do that

**Comment by Urhengulas** [comment]
> CI fails due to [la10736/rstest#169](https://github.com/la10736/rstest/issues/169) – I'm not sure why our nightly runs are blocking merges though, we might not want to do that

You are right, they shouldn't

**Comment by Urhengulas** [comment]
bors r=jonas-schievink

**Pull request merged**